src/contexts/Rewards/Provider.tsx

```diff
@@ -34,24 +42,42 @@ const Provider: React.FC = ({ children }) => {
     ethereum,
   }: { account: string | null | undefined; ethereum: provider } = useWallet()
 
+  //set proper contract address and merkle data depending on month
+  useEffect(() => {
+    switch (month) {
+      case 'December': {
+        break
+      }
+      case 'November': {
```

Comment:
This thing will start breaking after 1 year because we'll have duplicate months. Maybe include the year